RoddyCore/src/de/dkfz/roddy/execution/io/fs/FileSystemAccessProvider.groovy

```diff
@@ -769,8 +774,11 @@ public class FileSystemAccessProvider {
     }
 
     int getDefaultUserMask() {
```

Comment:
Here it is a "default" umask, but the variable is calles `_umask`. Make it consistent -- either with or without "default", whatever fits better.

Reply:
see above. Will rename it.